src/main/antora/modules/web/pages/servlets/servlets.adoc

```diff
@@ -1,8 +1,7 @@
-= Jakarta Servlet Technology
+= Jakarta Servlet
 
-include::ROOT:partial$not_updated.adoc[]
-
-Jakarta Servlet technology provides dynamic, user-oriented content in web applications using a request-response programming model.
+Jakarta Servlet is a corner stone web framework that can act as a presentation-oriented as well as a service-oriented web application.
+Jakarta Servlet intends to reduce the boilerplate code needed to convert the HTTP request into a Java object and to offer a Java object as a HTTP response, and to manage all the lifecycle around them.
```

Comment:
a HTTP -> an HTTP?

The H is pronounced as a vowel here.

Reply:
There were more occurrences https://github.com/jakartaee/jakartaee-tutorial/pull/36/commits/1d0f36f13a48dd67812d40f65501b0dfabf2b7db